composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/UserView.kt

```diff
@@ -1,58 +1,156 @@
+            Spacer(modifier = Modifier.height(36.dp))
+            Image(
+                modifier = Modifier
+                    .size(100.dp)
+                    .align(Alignment.CenterHorizontally),
+                imageVector = vectorResource(Res.drawable.profile_picture),
+                contentDescription = null
+            )
+            Spacer(modifier = Modifier.height(36.dp))
+            Column {
+                Text("Preferences")
+                ProfileButton(
+                    "Notifications",
+                    Res.drawable.notification,
+                    onClick = { /*TODO*/ })
+                ProfileButton(
+                    "Personal Information",
+                    Res.drawable.email,
+                    onClick = { /*TODO*/ })
+                ProfileButton(
+                    "Login & Security",
+                    Res.drawable.phone,
+                    onClick = { /*TODO*/ })
+                ProfileButton(
+                    "Rental History",
+                    Res.drawable.clock,
+                    onClick = { /*TODO*/ })
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                Text("Billing")
+                ProfileButton(
+                    "Payment Methods",
+                    Res.drawable.payment,
+                    onClick = { /*TODO*/ })
+                ProfileButton(
+                    "Billing Address",
+                    Res.drawable.location,
+                    onClick = { /*TODO*/ })
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                Text("Other")
+                ProfileButton(
+                    "Report a Problem",
+                    Res.drawable.report,
+                    onClick = { /*TODO*/ })
+                ProfileButton(
+                    "Security Policy",
+                    Res.drawable.security,
+                    onClick = { /*TODO*/ })
             }
 
-            for (i in roundedRating until 5) {
-                Image(
-                    modifier = Modifier.size(20.dp),
-                    imageVector = vectorResource(Res.drawable.star_outline),
-                    contentDescription = null
+            Spacer(modifier = Modifier.weight(1f))
+            Button(
+                onClick = { onLogOut() },
+                colors = ButtonDefaults.buttonColors(backgroundColor = Color(0xFFF9784B)),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(45.dp)
+                    .padding(2.dp)
+                    .shadow(4.dp, RoundedCornerShape(12.dp))
+                    .clip(RoundedCornerShape(12.dp))
+            ) {
+                Text(
+                    text = "Log Out",
+                    modifier = Modifier.padding(0.dp, 4.dp),
+                    color = Color.White
                 )
             }
         }
     }
 }
+
+@Composable
+fun ProfileButton(text: String, icon: DrawableResource, onClick: () -> Unit) {
```

Comment:
This might be more widely reusable - but can stay here for now and moved later if useful.

Reply:
Ya could be, the buttons on this page are slightly different than other pages so I just kept it here for now. I'll move it later if needed.